mnemonic/mnemonic.py

```diff
@@ -72,19 +79,19 @@ def __init__(self, language):
             )
 
     @staticmethod
-    def _get_directory():
+    def _get_directory() -> str:
         return os.path.join(os.path.dirname(__file__), "wordlist")
 
     @classmethod
-    def list_languages(cls):
+    def list_languages(cls) -> List[str]:
         return [
             f.split(".")[0]
             for f in os.listdir(cls._get_directory())
             if f.endswith(".txt")
         ]
 
     @staticmethod
-    def normalize_string(txt):
+    def normalize_string(txt: Union[bytes, str]) -> str:
```

Comment:
`AnyStr` instead of this union?

Reply:
This is something for another PR, and a breaking change, but we could also consider removing `bytes` support from `normalize_string` altogether. Internally we never pass `bytes` to this method.

Reply:
True, but it doesn't seem worth it. Variability here costs us next to nothing.